modules/ossm-rn-fixed-issues.adoc

```diff
@@ -14,7 +14,30 @@ Provide the following info for each issue if possible:
 *Result* - How has the behavior changed as a result? Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
 ////
 
-The following issue has been resolved in the current release:
+The following issues have been resolved in the current release:
+
+* https://issues.redhat.com/browse/OSSM-1397[OSSM-1397] Previously, if you removed the `maistra.io/member-of` label from a namespace, the {SMProductShortName} Operator did not not automatically reapply the label to the namespace. As a result, sidecar injection did not work in the namespace.
++
+The Operator would reapply the label to the namespace when you made changes to the `ServiceMeshMember` object, which triggered the reconciliation of this member object.
++
+Now, any change to the namespace also triggers the member object reconciliation.
+
+* https://issues.redhat.com/browse/OSSM-4851[OSSM-4851] Previously, when you attempted to create a new pod in a namespace that was part of a service mesh, the {SMProductShortName} Operator failed to create the pod. The Operator generated the following error message:
++
+`failed to run injection template: failed parsing generated injected YAML (check Istio sidecar injector configuration): unmarshal patched pod: json: cannot unmarshal string into Go struct field SecurityContext.spec.containers.securityContext.runAsGroup of type int64`
++
+Each time you create a new pod associated with a service mesh, the Operator creates a sidecar proxy using YAML template files. In the template files, if the `unAsGroup`, `runAsUser`, or `fsGroup` parameters were set to `nil` because of the current configuration that needed to be applied to the proxy, then the Operator could not deploy a new pod.  
```

Comment:
```suggestion
Each time you create a new pod associated with a service mesh, the Operator creates a sidecar proxy using YAML template files. In the template files, if the `runAsGroup`, `runAsUser`, or `fsGroup` parameters were set to `nil` because of the current configuration that needed to be applied to the proxy, then the Operator could not deploy a new pod.  
```

Reply:
done